packages/ag-charts-community/src/options/chart/gradientLegendOptions.ts

```diff
@@ -14,10 +31,15 @@ export interface AgGradientLegendOptions {
     gradient?: AgGradientLegendBarOptions;
     /** The spacing in pixels to use outside the legend. */
     spacing?: PixelSize;
-    /** Configuration for the legend gradient stops that consist of a color and a label. */
+    /**
+     * @deprecated
```

Comment:
Nit: Could you include a message after the `@deprecated` to guide the user towards the new way of doing this?